### PR TITLE
added interface for additional http headers

### PR DESCRIPTION
**Pull request by cmdrsabre**
now you can add additional http headers like authentication or accept
to the ImageManager, so all downloads use them

### CODE REVIEW

File: MapleBacon.xcodeproj/xcshareddata/xcschemes/MapleBacon.xcscheme [hunk 1]
```diff
@@ -62,6 +62,16 @@
                ReferencedContainer = "container:../../MapleBaconExample/MapleBaconExample.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
```
Comment (ramy-kfoury) [inline MapleBacon.xcodeproj/xcshareddata/xcschemes/MapleBacon.xcscheme]: is this change to the scheme needed?

**Comment by cmdrsabre** [comment]
Oh, definitely not. Was changed by my Xcode. I committed it
unintentionally. Sorry.

Ramy Kfoury <notifications@github.com> schrieb am So., 12. Feb. 2017, 13:38:

File: Library/MapleBacon/MapleBacon/ImageDownloadOperation.swift [hunk 2]
```diff
@@ -20,6 +20,8 @@ public final class ImageDownloadOperation: Operation {
     fileprivate let progress: Progress = Progress()
 
     public var completionHandler: ImageDownloaderCompletion?
+    
+    public var httpAdditionalHeaders:[AnyHashable: Any]?
```
Comment (ramy-kfoury) [inline Library/MapleBacon/MapleBacon/ImageDownloadOperation.swift]: can you fix the spaces here please: `public var httpAdditionalHeaders: [AnyHashable: Any]?`

File: Library/MapleBacon/MapleBacon/ImageManager.swift [hunk 3]
```diff
@@ -11,6 +11,8 @@ public final class ImageManager {
     fileprivate let downloadQueue = OperationQueue()
     fileprivate var downloadsInProgress = [URL: ImageDownloadOperation]()
 
+    public var httpAdditionalHeaders:[AnyHashable:Any]?
```
Comment (ramy-kfoury) [inline Library/MapleBacon/MapleBacon/ImageManager.swift]: and here as well, same as above, spaces

File: MapleBacon.xcodeproj/xcshareddata/xcschemes/MapleBacon.xcscheme (hunk 1 above)
Comment (ramy-kfoury) [inline MapleBacon.xcodeproj/xcshareddata/xcschemes/MapleBacon.xcscheme]: will you remove this from the scheme?

**Comment by ramy-kfoury** [comment]
great, thanks @cmdrsabre for the change

**Pull request merged**